oem.json

```diff
@@ -592,6 +592,13 @@
         "description": "Some kind of theme overlay for Nokia devices?\nSome users claim to not see any differences when removed.",
         "removal": "caution"
     },
+    {
+        "id": "com.dti.lenovo.tablet",
+        "label": "Mobile Services",
+        "description": "Silently installs unwanted packages without user permission.\n",
+        "web": "https://www.andrewnile.co.uk/blog/lenovo-mobile-services/.\nPithus report https://beta.pithus.org/report/4ebcc8b5a13851054d85153b0d4086e4b2b5adaee66eaea5843078c6134e9dd7",
```

Comment:
Web takes array of websites.

e.g.

```json
"web": [
  "https://www.andrewnile.co.uk/blog/lenovo-mobile-services/",
  "https://beta.pithus.org/report/4ebcc8b5a13851054d85153b0d4086e4b2b5adaee66eaea5843078c6134e9dd7"
]
```

Reply:
So other texts are not allowed for `web`? ~Pithus report~
Please fix it from your side, I'll follow the same format next time.
Also request to add an [example](https://github.com/0x192/universal-android-debloater/wiki#how-to-contribute) in README for non-technical users like me. Thanks.

Reply:
> So other texts are not allowed for `web`? ~Pithus report~

No. It already includes pithus.org, adding **Pithus report** is redundant anyway.